cs-serviceclient-sample/cs-serviceclient-sample/MVVM/Model/FPlusCommunicator.cs

```diff
@@ -102,9 +105,11 @@ private void MessageReceived(IMqttClient mqttClient, Payload payload, MqttApplic
             {
                 return;
             }
-            Debug.WriteLine($"Message received on: {mqttMessageObject.Topic}");
-            Debug.WriteLine(payload.Metrics.Count);
-            Debug.WriteLine(payload.Metrics);
+            // Debug.WriteLine($"Message received on: {mqttMessageObject.Topic}");
+            // Debug.WriteLine(payload.Metrics.Count);
+            // Debug.WriteLine(payload.Metrics);
+            
+            DistributeMessage?.Invoke(mqttMessageObject, payload);
```

Comment:
Do we need to distribute the raw mqttMessage

Reply:
Maybe not the whole thing but that's where the topic lives